server/boot/user.js

```diff
@@ -631,4 +643,61 @@ module.exports = function(app) {
       return res.render('account/forgot');
     });
   }
+
+  function getReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    const { user } = req;
+    if (!user.email || !user.emailVerified) {
+      req.flash('error', {
+        msg: 'We do not have your verified email address on record, '
+        + 'please add it in the settings to continue with your request.'
+      });
+      return res.redirect('/settings');
+    }
+
+    return res.render('account/report-profile', {
+      title: 'Report User',
+      username
+    });
+  }
+
+  function postReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    const report = req.body.reportDescription;
+    const { user } = req;
+
+    debug(report);
+
+    return Email.send$({
+      type: 'email',
+      to: 'Team@FreeCodeCamp.com',
+      cc: user.email,
+      from: 'Team@FreeCodeCamp.com',
+      subject: 'Abuse Report : Reporting ' + username + '\'s profile.',
+      text: dedent(`
+        Hello Team,\n
+        This is to report the profile of ${username}.\n
+        Report Details:\n
+        ${report}\n\n
+        Reported by:
+        Username: ${user.username}
+        Name: ${user.name}
+        Email: ${user.email}\n
+        Thanks and regards,
+        ${user.name}
+      `)
+    }, function(err) {
+      if (err) {
+        req.flash('errors', err.message);
+        return res.redirect('/');
+      }
+
+      req.flash('info', {
+        msg: 'An e-mail has been sent to ' +
+        user.email +
```

Comment:
We are showing a notif to the reporter. This is the reporter's email.

We are keeping them in the copy to continue the follow-up, plus having them in copy means they won't be spamming with multiple reports.